README.md

```diff
@@ -114,6 +119,21 @@ Would filter your list of articles to any article with "cakephp" in the `title`
 or `content` field. You might choose to make a `get` form which posts the filter
 directly to the url, or create links manually.
 
+### Creating your GET form
+You can change the method of a form when you create it.
+
+```php
+    echo $this->Form->create(null, ['type' => 'get']);
```

Comment:
If you use GET it will bypass the PRG process. Also you will end up with the token fields in query string if using security component. So the form should use POST.